test/known_issues/test-vm-defineProperty-setter.js

```diff
@@ -0,0 +1,25 @@
+// Refs:https://github.com/nodejs/node/pull/38557
```

Comment:
Since the behavior is being fixed, the test should be located in the `test/parallel` not `test/known_issues`